nocasedict/_nocasedict.py

```diff
@@ -417,7 +458,7 @@ def keys(self):
         # pylint: disable=line-too-long
         """
         Return a view on (in Python 3) or a list of (in Python 2) the
-        dictionary keys (in the original lexical case) in the preserved order.
+        dictionary keys (in the original lexical case) in dictionary order.
```

Comment:
I am not sure what "dictionary order" means

Reply:
DONE - Sorry. I meant to say "dictionary iteration order". Changed everywhere.